local_costmap:
   global_frame: /world
   robot_base_frame: base_link #vehicle
   update_frequency: 5.0
   publish_frequency: 2.0
   static_map: false
   rolling_window: true
   width: 8.0
   height: 8.0
   resolution: 0.05
   transform_tolerance: 0.5
